selfdrive/test/ci_shell.sh

```diff
@@ -11,6 +11,7 @@ fi
 
 docker run \
        -it \
+       --shm-size=100m \
```

Comment:
I had to increase the memory usage due to getting a `Fatal Python error: Bus error`

https://github.com/commaai/openpilot/actions/runs/9201668216/job/25310251273